install/kubernetes/cilium/templates/cilium-operator/clusterrole.yaml

```diff
@@ -95,6 +95,12 @@ rules:
   - delete
   - patch
 {{- end }}
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - '*'
```

Comment:
Indeed only patch create seems to be needed :+1:, and I noticed I added this change in the wrong commit :see_no_evil:, it's now part of the helm specific commit with a small explanation in the description :D (that the endpointslice controller sends events in case of error basically)